tests/test_ui_manager.py

```diff
@@ -242,7 +244,8 @@ def test_add_font_paths_and_preload_fonts(self, _init_pygame, default_ui_manager
         We sets the path to a font, preload it then try and use it in a text box and see if any errors or warnings
         happen.
         """
-        default_ui_manager.add_font_paths(font_name='roboto', regular_path='tests/data/Roboto-Regular.ttf')
+        default_ui_manager.add_font_paths(font_name='roboto',
+                                          regular_path=os.path.join('data', 'Roboto-Regular.ttf'))
```

Comment:
This is mostly a good change, as joined paths are more robust - but as above excluding the `tests` part from the path will only work when running the tests from inside the `tests` subdirectory when I think they are generally run on the CI with the root pygame_gui directory as the working directory.